### PR TITLE
Add audit of gemfile

### PR DESCRIPTION
**Pull request by jcoyne**
Prints this report at the end:
```
Errors found:
sul-dlss/common-accessioning: CVE-2020-8184
sul-dlss/dor-services-app: CVE-2020-8184

Summary:
NAME: rack
ADVISORY: CVE-2020-8184
CRITICALITY: Unknown
VERSION: 2.2.2
URL: https://groups.google.com/g/rubyonrails-security/c/OWtmozPH9Ak
TITLE: Percent-encoded cookies can be used to overwrite existing prefixed cookie names
SOLUTION: upgrade to ~> 2.1.4, >= 2.2.3
```

### CODE REVIEW

**Comment by mjgiarlo** [comment]
We should show our A-Team colleagues this.

File: deploy.rb [hunk 1]
```diff
@@ -8,6 +9,7 @@
 # ./deploy.rb stage
 
 require 'fileutils'
+require 'auditor'
```
Comment (mjgiarlo) [inline deploy.rb:12]: @jcoyne Should line 40 below also include some logic to set `SKIP_BUNDLE_AUDIT` in the IO env? That way we only run the bundle audit when we want to.

Comment (mjgiarlo) [reply deploy.rb]: done.

**Comment by ndushay** [comment]
I really like this with the additional changes.  W00t!

**Pull request merged**